sdcm/nemesis.py

```diff
@@ -4491,11 +4493,11 @@ def _switch_to_network_replication_strategy(self, keyspaces: List[str]) -> None:
                 # no need to switch as already is NetworkTopology
                 continue
             self.log.info(f"Switching replication strategy to Network for '{keyspace}' keyspace")
-            if keyspace == "system_auth" and replication_strategy.replication_factor != len(nodes_by_region[region]):
+            if keyspace == "system_auth" and replication_strategy.replication_factors[0] != len(nodes_by_region[region]):  # pylint: disable=protected-access
```

Comment:
is `# pylint: disable=protected-access` required here and below?

Reply:
removed